GettingStarted.md

```diff
@@ -1,80 +1,165 @@
-This document describes how to prepare your work envionment to use the HoloToolkit-Unity in your project.
+This document describes how to prepare your work environment to use MixedRealityToolkit-Unity in your Unity 3D project.
 
-# 1. Setting up your development environment
+# 1. Check your Windows 10 version
+It is recommended to be running the Windows 10 Fall Creators update for modern Windows Universal apps targeting Mixed Reality Headsets
+
+This can be verified by running the "WinVer" application from the Windows Run command
+
+`Windows Key + R -> WinVer`
+
+Which will display a new window as follows:
+
+![Windows Version dialog](/External/ReadMeImages/WindowsVersionFCU.png)
+
+If you are not running the Windows 10 Fall Creators update, then you will need to Update your version of Windows.
+
+# 2. Setting up your development environment
 Be sure to enable Developer mode for Windows 10 via:
 
 `Action Center -> All Settings -> Update & Security -> For Developers -> Enable Developer mode`
 
 ![Enable Developer Mode](/External/ReadMeImages/EnableDevModeWin10.PNG "Enable Developer Mode for Windows 10")
 
-Be sure to download and install [Visual Studio 2017](https://www.visualstudio.com/vs/) and these required components:
+If you have not already, download and install [Visual Studio 2017](https://www.visualstudio.com/vs/) and these required components:
 
 ![Visual Studio Components](/External/ReadMeImages/VisualStudioComponents.PNG)
```

Comment:
We should probably update this screenshot to include msbuild and the updated SDKs

Reply:
That is a good point, firing up the VS Installer